DMTN-049.tex

```diff
@@ -123,14 +136,21 @@ \subsection{Federating a User-Generated Photo-$z$ Catalog}\label{ssec:time_ops_u
 
 \subsection{International Partnerships (In-Kind Contributions)}\label{ssec:time_inkind}
 
-All in-kind program contributions related to photo-$z$ will be subject to the processes outlined in this roadmap and will be included and integrated into the communal efforts to optimize the LSST photo-$z$.
-These contributions might include, e.g., individuals' expertise; scientific analyses; software algorithms, tools, or infrastructure; computational resources; datasets or observing time.
-These contributions might be directed to the Project (e.g., DM, commissioning team), and/or to the science community (e.g., a Science Collaboration, all data rights holders). 
+As the process for International Partnerships with Rubin Observatory via in-kind contributions evolved, photometric redshifts emerged as an area in which (1) multiple teams propose to provide different types of contributions and (2) optimizing the LSST scientific returns on these contributions requires the coordination across the Rubin community: the in-kind teams, Rubin staff, the Science Collaborations, and the broad community of data rights holders.
```

Comment:
" requires coordination across the Rubin community:"

Reply:
fixed